erpnext/accounts/report/trial_balance_for_party/trial_balance_for_party.py

```diff
@@ -19,7 +19,7 @@ def execute(filters=None):
 	return columns, data
 	
 def get_data(filters, show_party_name):
-	party_name_field = "customer_name" if filters.get("party_type")=="Customer" else "supplier_name"
+	party_name_field = "customer_name" if filters.get("party_type")=="Customer" else "supplier_name" if filters.get("party_type")=="Supplier" else "Employee"
```

Comment:
Please add a multi-line `if` statement since there are more than one ifs, or make a dict of fieldnames and use the correct fieldname.